src/components/modals/SendModal.vue

```diff
@@ -141,6 +151,7 @@
                     <LabelInput
                         v-model="message"
                         :placeholder="$t('Add a public message...')"
+                        :disabled="!isValidRecipient"
                         :maxBytes="64"
```

Comment:
Instead of disabling the `message` LabelInput, the whole section should be hidden, just like the `amount-section`.